app/models/form/sales/questions/buyer1_income_value_check.rb

```diff
@@ -3,7 +3,7 @@ def initialize(id, hsh, page)
     super
     @id = "income1_value_check"
     @check_answer_label = "Income confirmation"
-    @header = "Are you sure this income is correct?"
+    @header = "Are you sure this is correct?"
```

Comment:
I think this is less clear as slightly more ambiguous what "this" refers to in the context of the message above it